templates/embeds/collapsible.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@Tabler/embeds/card.html.twig' %}
+{% set collapsed = false %}
```

Comment:
Is this still needed?

Reply:
Yes, The should always be collapsed, and then you define through the accordionItem `open` flag if the content is visible or not.